pinning new sha & version

BubblyOrca opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Commit:
pinning new sha & version

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

Commit:
removed old maintainers

Commit:
pinning correct sha

Commit:
pinning new pytorch version: 1.8 in run

Commit:
pinning dependencies:- tabulate
    - iopath >=0.1.7,<0.1.10
    - dataclasses; python_version <3.7
    - omegaconf >=2.1
    - hydra-core >=1.1
    - black ==21.4b2

Commit:
specifying dataclasses

Commit:
unpinning  - tabulate
     - iopath >=0.1.7,<0.1.10
     - dataclasses; python_version <3.7
     - omegaconf >=2.1
     - hydra-core >=1.1
     - black ==21.4b2

Commit:
pinning pycocotools>=2.0.2

Commit:
specified python version

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Non noarch packages should have python requirement without any version constraints.
 * Non noarch packages should have python requirement without any version constraints.

Commit:
unpinned python version & specified fvcore >=0.1.5,<0.1.6

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

Commit:
pinned libgcc-ng >=9.4.0

Commit:
unpinned libgcc-ng version

Commit:
reset build number to 0

Commit:
unpinned fvcore >=0.1.5,<0.1.6 (version only)

Commit:
changed version: yacs >=0.1.8

Commit:
repinned yac>=0.1.6

Commit:
pinned libgcc-ng

Commit:
pinned libgcc instead of libgcc-ng

Commit:
pinned ca-certificates

Commit:
unpinned libgcc& ca-certificates

Commit:
unpinned pytorch version

Commit:
pinned pytorch version back

baszalmstra commented:
Closing this because the feedstock already builds 0.6
